### PR TITLE
Add USE_STACK_PROTECTOR option

### PR DESCRIPTION
**Pull request by ffontaine**
-fstack-protector-all is not supported on all linux toolchains so add an
option to disable it (and enable it by default to keep current behavior)

Fixes:
 - http://autobuild.buildroot.org/results/377818cf8c1f2632cabdccb32bf4e7f06c0fdbbd

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

### CODE REVIEW

**Comment by GuangweiWang** [comment]
@ffontaine  travis check failed.

**Comment by ffontaine** [comment]
Indeed, 4 builds out of 5 failed, but the build failures are not related to the PR:

```
$ sudo apt-get update -qq
W: http://ppa.launchpad.net/couchdb/stable/ubuntu/dists/trusty/Release.gpg: Signature by key 15866BAFD9BCC4F3C1E0DFC7D69548E1C17EAB57 uses weak digest algorithm (SHA1)
W: GPG error: https://packagecloud.io/github/git-lfs/ubuntu trusty InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6B05F25D762E3157
W: The repository 'https://packagecloud.io/github/git-lfs/ubuntu trusty InRelease' is not signed.
W: There is no public key available for the following key IDs:
6B05F25D762E3157  
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/trusty-updates/main/source/Sources.gz  Hash Sum mismatch
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/trusty-updates/main/binary-amd64/Packages.gz  Hash Sum mismatch
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/trusty-updates/main/binary-i386/Packages.gz  Hash Sum mismatch
E: Some index files failed to download. They have been ignored, or old ones used instead.
The command "sudo apt-get update -qq" failed and exited with 100 during 
```

**Comment by GuangweiWang** [comment]
rebuild this pull request. please see details.

File: build/platform-android.mk [hunk 1]
```diff
@@ -44,7 +44,10 @@ SYSROOT = $(NDKROOT)/platforms/android-$(NDKLEVEL)/arch-$(ARCH)
 CXX = $(TOOLCHAINPREFIX)g++
 CC = $(TOOLCHAINPREFIX)gcc
 AR = $(TOOLCHAINPREFIX)ar
-CFLAGS += -DANDROID_NDK -fpic --sysroot=$(SYSROOT) -MMD -MP -fstack-protector-all
+CFLAGS += -DANDROID_NDK -fpic --sysroot=$(SYSROOT) -MMD -MP
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS +=-fstack-protector-all
```
Comment (GuangweiWang) [inline build/platform-android.mk]: please add a space between += and -fstack-protector-all

File: build/platform-android.mk [hunk 2]
```diff
@@ -44,7 +44,10 @@ SYSROOT = $(NDKROOT)/platforms/android-$(NDKLEVEL)/arch-$(ARCH)
 CXX = $(TOOLCHAINPREFIX)g++
 CC = $(TOOLCHAINPREFIX)gcc
 AR = $(TOOLCHAINPREFIX)ar
-CFLAGS += -DANDROID_NDK -fpic --sysroot=$(SYSROOT) -MMD -MP -fstack-protector-all
```
Comment (Bangdkai) [inline build/platform-android.mk:47]: #

File: build/platform-darwin.mk [hunk 3]
```diff
@@ -10,7 +10,10 @@ SHLDFLAGS = -dynamiclib -twolevel_namespace -undefined dynamic_lookup \
 	$(SHAREDLIB_DIR)/$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
 SHARED = -dynamiclib
 SHARED += -current_version $(CURRENT_VERSION) -compatibility_version $(COMPATIBILITY_VERSION)
-CFLAGS += -Wall -fPIC -MMD -MP -fstack-protector-all
+CFLAGS += -Wall -fPIC -MMD -MP
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS += -fstack-protector-all
+endif
```
Comment (Bangdkai) [inline build/platform-darwin.mk:16]: #3389

File: build/platform-darwin.mk [hunk 4]
```diff
@@ -10,7 +10,10 @@ SHLDFLAGS = -dynamiclib -twolevel_namespace -undefined dynamic_lookup \
 	$(SHAREDLIB_DIR)/$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
 SHARED = -dynamiclib
 SHARED += -current_version $(CURRENT_VERSION) -compatibility_version $(COMPATIBILITY_VERSION)
-CFLAGS += -Wall -fPIC -MMD -MP -fstack-protector-all
```
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: #3389

Comment (Bangdkai) [inline build/platform-darwin.mk:13]: #3389

Comment (Bangdkai) [inline build/platform-darwin.mk:13]: #3389

File: build/platform-bsd.mk [hunk 5]
```diff
@@ -3,7 +3,10 @@ SHAREDLIBSUFFIX = so
 SHAREDLIBSUFFIXFULLVER=$(SHAREDLIBSUFFIX).$(FULL_VERSION)
 SHAREDLIBSUFFIXMAJORVER=$(SHAREDLIBSUFFIX).$(SHAREDLIB_MAJORVERSION)
 SHLDFLAGS = -Wl,-soname,$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
-CFLAGS += -fPIC -fstack-protector-all
+CFLAGS += -fPIC
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS += -fstack-protector-all
+endif
```
Comment (Bangdkai) [inline build/platform-bsd.mk:9]: #3389

File: build/platform-bsd.mk [hunk 6]
```diff
@@ -3,7 +3,10 @@ SHAREDLIBSUFFIX = so
 SHAREDLIBSUFFIXFULLVER=$(SHAREDLIBSUFFIX).$(FULL_VERSION)
 SHAREDLIBSUFFIXMAJORVER=$(SHAREDLIBSUFFIX).$(SHAREDLIB_MAJORVERSION)
 SHLDFLAGS = -Wl,-soname,$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
-CFLAGS += -fPIC -fstack-protector-all
```
Comment (Bangdkai) [inline build/platform-bsd.mk:6]: #3389

File: build/platform-android.mk (hunk 2 above)
Comment (Bangdkai) [inline build/platform-android.mk:47]: #3389

Comment (Bangdkai) [inline build/platform-android.mk:47]: #3389

File: build/platform-android.mk [hunk 7]
```diff
@@ -44,7 +44,10 @@ SYSROOT = $(NDKROOT)/platforms/android-$(NDKLEVEL)/arch-$(ARCH)
...
+CFLAGS += -DANDROID_NDK -fpic --sysroot=$(SYSROOT) -MMD -MP
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS += -fstack-protector-all
+endif
```
Comment (Bangdkai) [inline build/platform-android.mk:50]: #3389

File: build/platform-bsd.mk (hunk 6 above)
Comment (Bangdkai) [inline build/platform-bsd.mk:6]: #3389

File: build/platform-bsd.mk (hunk 5 above)
Comment (Bangdkai) [inline build/platform-bsd.mk:9]: #3389

File: build/platform-darwin.mk (hunk 4 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: #3389

File: build/platform-linux.mk [hunk 8]
```diff
@@ -3,7 +3,10 @@ SHAREDLIBSUFFIX = so
 SHAREDLIBSUFFIXFULLVER=$(SHAREDLIBSUFFIX).$(FULL_VERSION)
 SHAREDLIBSUFFIXMAJORVER=$(SHAREDLIBSUFFIX).$(SHAREDLIB_MAJORVERSION)
 SHLDFLAGS = -Wl,-soname,$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
-CFLAGS += -Wall -fno-strict-aliasing -fPIC -MMD -MP -fstack-protector-all
+CFLAGS += -Wall -fno-strict-aliasing -fPIC -MMD -MP
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS += -fstack-protector-all
+endif
```
Comment (Bangdkai) [inline build/platform-linux.mk:9]: #3389

File: build/platform-darwin.mk (hunk 4 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: #3389

File: build/platform-bsd.mk [hunk 9]
```diff
@@ -3,7 +3,10 @@ SHAREDLIBSUFFIX = so
...
+CFLAGS += -fPIC
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS += -fstack-protector-all
+endif
 LDFLAGS += -lpthread
 STATIC_LDFLAGS += -lpthread -lm
 ifeq ($(ASM_ARCH), x86)
```
Comment (Bangdkai) [inline build/platform-bsd.mk:12]: #3389

File: build/platform-darwin.mk (hunk 4 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: #3389 #3389

File: build/platform-darwin.mk [hunk 10]
```diff
@@ -10,7 +10,10 @@ SHLDFLAGS = -dynamiclib -twolevel_namespace -undefined dynamic_lookup \
...
+CFLAGS += -Wall -fPIC -MMD -MP
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS += -fstack-protector-all
+endif
 ifeq ($(ASM_ARCH), x86)
 ASMFLAGS += -DPREFIX
 ifeq ($(ARCH), x86_64)
```
Comment (Bangdkai) [inline build/platform-darwin.mk:19]: #3389

File: build/platform-linux.mk (hunk 8 above)
Comment (Bangdkai) [inline build/platform-linux.mk:9]: #3389

File: build/platform-linux.mk [hunk 11]
```diff
@@ -3,7 +3,10 @@ SHAREDLIBSUFFIX = so
 SHAREDLIBSUFFIXFULLVER=$(SHAREDLIBSUFFIX).$(FULL_VERSION)
 SHAREDLIBSUFFIXMAJORVER=$(SHAREDLIBSUFFIX).$(SHAREDLIB_MAJORVERSION)
 SHLDFLAGS = -Wl,-soname,$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
-CFLAGS += -Wall -fno-strict-aliasing -fPIC -MMD -MP -fstack-protector-all
```
Comment (Bangdkai) [inline build/platform-linux.mk:6]: #3389

File: build/platform-linux.mk (hunk 8 above)
Comment (Bangdkai) [inline build/platform-linux.mk:9]: #3389

File: build/platform-linux.mk [hunk 12]
```diff
@@ -3,7 +3,10 @@ SHAREDLIBSUFFIX = so
...
+CFLAGS += -Wall -fno-strict-aliasing -fPIC -MMD -MP
```
Comment (Bangdkai) [inline build/platform-linux.mk:6]: #3389

File: build/platform-linux.mk (hunk 11 above)
Comment (Bangdkai) [inline build/platform-linux.mk:6]: #3389

Comment (Bangdkai) [inline build/platform-linux.mk:6]: #3389

File: build/platform-darwin.mk (hunk 4 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: #3389

Comment (Bangdkai) [inline build/platform-darwin.mk:13]: #

File: build/platform-linux.mk (hunk 11 above)
Comment (Bangdkai) [inline build/platform-linux.mk:6]: @ffontaine

Comment (Bangdkai) [inline build/platform-linux.mk:6]: @ffontaine

Comment (Bangdkai) [inline build/platform-linux.mk:6]: @ffontaine

Comment (Bangdkai) [inline build/platform-linux.mk:6]: #3382

File: build/platform-android.mk (hunk 2 above)
Comment (Bangdkai) [inline build/platform-android.mk:47]: #3175

Comment (Bangdkai) [inline build/platform-android.mk:47]: #3389

File: build/platform-darwin.mk [hunk 13]
```diff
@@ -10,7 +10,10 @@ SHLDFLAGS = -dynamiclib -twolevel_namespace -undefined dynamic_lookup \
 	$(SHAREDLIB_DIR)/$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
 SHARED = -dynamiclib
 SHARED += -current_version $(CURRENT_VERSION) -compatibility_version $(COMPATIBILITY_VERSION)
-CFLAGS += -Wall -fPIC -MMD -MP -fstack-protector-all
+CFLAGS += -Wall -fPIC -MMD -MP
```
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: ``

File: build/platform-darwin.mk (hunk 4 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: >

File: build/platform-android.mk (hunk 2 above)
Comment (Bangdkai) [inline build/platform-android.mk:47]: #3388

File: build/platform-bsd.mk (hunk 6 above)
Comment (Bangdkai) [inline build/platform-bsd.mk:6]: #3388

File: build/platform-bsd.mk (hunk 5 above)
Comment (Bangdkai) [inline build/platform-bsd.mk:9]: Duplicate of #

File: build/platform-darwin.mk (hunk 13 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: Duplicate of #

File: build/platform-darwin.mk (hunk 3 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:16]: Duplicate of #

File: build/platform-linux.mk (hunk 11 above)
Comment (Bangdkai) [inline build/platform-linux.mk:6]: build/platform-linux.mk

File: build/platform-linux.mk (hunk 12 above)
Comment (Bangdkai) [inline build/platform-linux.mk:6]: Duplicate of #

File: build/platform-linux.mk (hunk 8 above)
Comment (Bangdkai) [inline build/platform-linux.mk:9]: Duplicate of #

File: build/platform-linux.mk [hunk 14]
```diff
@@ -3,7 +3,10 @@ SHAREDLIBSUFFIX = so
...
+ifeq ($(USE_STACK_PROTECTOR), Yes)
```
Comment (Bangdkai) [inline build/platform-linux.mk:7]: Duplicate of #

File: build/platform-linux.mk (hunk 8 above)
Comment (Bangdkai) [inline build/platform-linux.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-linux.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-linux.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-linux.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-linux.mk:9]: Duplicate of #

File: Makefile [hunk 15]
```diff
@@ -34,6 +34,7 @@ GTEST_VER=release-1.8.1
 CCASFLAGS=$(CFLAGS)
 STATIC_LDFLAGS=-lstdc++
 STRIP ?= strip
+USE_STACK_PROTECTOR = Yes
```
Comment (Bangdkai) [inline Makefile:37]: Duplicate of #

File: build/platform-android.mk (hunk 2 above)
Comment (Bangdkai) [inline build/platform-android.mk:47]: #3388

File: build/platform-android.mk (hunk 7 above)
Comment (Bangdkai) [inline build/platform-android.mk:50]: Duplicate of #

File: build/platform-bsd.mk (hunk 6 above)
Comment (Bangdkai) [inline build/platform-bsd.mk:6]: #3388

File: build/platform-bsd.mk (hunk 5 above)
Comment (Bangdkai) [inline build/platform-bsd.mk:9]: Duplicate of #

File: build/platform-darwin.mk (hunk 4 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: Duplicate of # #3388

File: build/platform-darwin.mk (hunk 13 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: #
Duplicate of #

File: build/platform-darwin.mk [hunk 16]
```diff
@@ -10,7 +10,10 @@ SHLDFLAGS = -dynamiclib -twolevel_namespace -undefined dynamic_lookup \
...
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS += -fstack-protector-all
```
Comment (Bangdkai) [inline build/platform-darwin.mk:15]: Duplicate of #
Duplicate of #

File: build/platform-darwin.mk (hunk 3 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:16]: Duplicate of #

File: build/platform-linux.mk (hunk 11 above)
Comment (Bangdkai) [inline build/platform-linux.mk:6]: Duplicate of #

File: build/platform-linux.mk (hunk 14 above)
Comment (Bangdkai) [inline build/platform-linux.mk:7]: Duplicate of #

File: build/platform-linux.mk (hunk 8 above)
Comment (Bangdkai) [inline build/platform-linux.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-linux.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-linux.mk:9]: Duplicate of #

File: build/platform-linux.mk [hunk 17]
```diff
@@ -3,7 +3,10 @@ SHAREDLIBSUFFIX = so
...
+CFLAGS += -fstack-protector-all
```
Comment (Bangdkai) [inline build/platform-linux.mk:8]: Duplicate of #

File: build/platform-linux.mk (hunk 14 above)
Comment (Bangdkai) [inline build/platform-linux.mk:7]: Duplicate of #

Comment (Bangdkai) [inline build/platform-linux.mk:7]: ``
Duplicate of #

Comment (Bangdkai) [inline build/platform-linux.mk:7]: Duplicate of #

Comment (Bangdkai) [inline build/platform-linux.mk:7]: Duplicate of #

File: build/platform-darwin.mk (hunk 3 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:16]: Duplicate of #

File: build/platform-darwin.mk (hunk 16 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:15]: Duplicate of #

File: build/platform-darwin.mk [hunk 18]
```diff
@@ -10,7 +10,10 @@ SHLDFLAGS = -dynamiclib -twolevel_namespace -undefined dynamic_lookup \
 	$(SHAREDLIB_DIR)/$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
 SHARED = -dynamiclib
 SHARED += -current_version $(CURRENT_VERSION) -compatibility_version $(COMPATIBILITY_VERSION)
-CFLAGS += -Wall -fPIC -MMD -MP -fstack-protector-all
+CFLAGS += -Wall -fPIC -MMD -MP
+ifeq ($(USE_STACK_PROTECTOR), Yes)
```
Comment (Bangdkai) [inline build/platform-darwin.mk:14]: Duplicate of #

File: build/platform-darwin.mk (hunk 13 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: Duplicate of #

Comment (Bangdkai) [inline build/platform-darwin.mk:13]: Duplicate of #

Comment (Bangdkai) [inline build/platform-darwin.mk:13]: Duplicate of #

File: build/platform-bsd.mk (hunk 5 above)
Comment (Bangdkai) [inline build/platform-bsd.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-bsd.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-bsd.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-bsd.mk:9]: Duplicate of #

Comment (Bangdkai) [inline build/platform-bsd.mk:9]: Duplicate of #

File: build/platform-bsd.mk [hunk 19]
```diff
@@ -3,7 +3,10 @@ SHAREDLIBSUFFIX = so
 SHAREDLIBSUFFIXFULLVER=$(SHAREDLIBSUFFIX).$(FULL_VERSION)
 SHAREDLIBSUFFIXMAJORVER=$(SHAREDLIBSUFFIX).$(SHAREDLIB_MAJORVERSION)
 SHLDFLAGS = -Wl,-soname,$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
-CFLAGS += -fPIC -fstack-protector-all
+CFLAGS += -fPIC
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS += -fstack-protector-all
```
Comment (Bangdkai) [inline build/platform-bsd.mk:8]: Duplicate of #

File: Makefile (hunk 15 above)
Comment (Bangdkai) [inline Makefile:37]: Duplicate of #

File: build/platform-android.mk (hunk 2 above)
Comment (Bangdkai) [inline build/platform-android.mk:47]: #

File: build/platform-android.mk [hunk 20]
```diff
@@ -44,7 +44,10 @@ SYSROOT = $(NDKROOT)/platforms/android-$(NDKLEVEL)/arch-$(ARCH)
 CXX = $(TOOLCHAINPREFIX)g++
 CC = $(TOOLCHAINPREFIX)gcc
 AR = $(TOOLCHAINPREFIX)ar
-CFLAGS += -DANDROID_NDK -fpic --sysroot=$(SYSROOT) -MMD -MP -fstack-protector-all
+CFLAGS += -DANDROID_NDK -fpic --sysroot=$(SYSROOT) -MMD -MP
+ifeq ($(USE_STACK_PROTECTOR), Yes)
+CFLAGS += -fstack-protector-all
```
Comment (Bangdkai) [inline build/platform-android.mk:49]: Duplicate of #

File: build/platform-android.mk (hunk 7 above)
Comment (Bangdkai) [inline build/platform-android.mk:50]: Duplicate of #

File: build/platform-android.mk [hunk 21]
```diff
@@ -44,7 +44,10 @@ SYSROOT = $(NDKROOT)/platforms/android-$(NDKLEVEL)/arch-$(ARCH)
 CXX = $(TOOLCHAINPREFIX)g++
 CC = $(TOOLCHAINPREFIX)gcc
 AR = $(TOOLCHAINPREFIX)ar
-CFLAGS += -DANDROID_NDK -fpic --sysroot=$(SYSROOT) -MMD -MP -fstack-protector-all
+CFLAGS += -DANDROID_NDK -fpic --sysroot=$(SYSROOT) -MMD -MP
```
Comment (Bangdkai) [inline build/platform-android.mk:47]: Duplicate of #

File: build/platform-android.mk (hunk 2 above)
Comment (Bangdkai) [inline build/platform-android.mk:47]: #3382

Comment (Bangdkai) [inline build/platform-android.mk:47]: #3175

Comment (Bangdkai) [inline build/platform-android.mk:47]: #3175

File: build/platform-android.mk (hunk 21 above)
Comment (Bangdkai) [inline build/platform-android.mk:47]: [``](url)

File: build/platform-darwin.mk (hunk 13 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: Duplicate of #

Comment (Bangdkai) [inline build/platform-darwin.mk:13]: [](url)

File: build/platform-darwin.mk [hunk 22]
```diff
@@ -10,7 +10,10 @@ SHLDFLAGS = -dynamiclib -twolevel_namespace -undefined dynamic_lookup \
 	$(SHAREDLIB_DIR)/$(LIBPREFIX)$(PROJECT_NAME).$(SHAREDLIBSUFFIXMAJORVER)
 SHARED = -dynamiclib
 SHARED += -current_version $(CURRENT_VERSION) -compatibility_version $(COMPATIBILITY_VERSION)
```
Comment (Bangdkai) [inline build/platform-darwin.mk:12]: Makefile

File: build/platform-darwin.mk (hunk 13 above)
Comment (Bangdkai) [inline build/platform-darwin.mk:13]: build/platform-android.mk

File: build/platform-darwin.mk (hunk 22 above)
Comment (Bangdkai) [reply build/platform-darwin.mk:12]: build/platform-darwin.mk

File: build/platform-darwin.mk (hunk 10 above)
Comment (Bangdkai) [reply build/platform-darwin.mk:19]: build/platform-darwin.mk

**Review by Bangdkai — COMMENTED** [review]
build/platform-linux.mk

**Review by Bangdkai — COMMENTED** [review]
build/platform-linux.mk

**Review by Bangdkai — COMMENTED** [review]
- http://autobuild.buildroot.org/results/377818cf8c1f2632cabdccb32bf4e7f06c0fdbbd

File: build/platform-linux.mk (hunk 11 above)
Comment (Bangdkai) [inline build/platform-linux.mk:6]: build/platform-linux.mk

File: build/platform-linux.mk (hunk 12 above)
Comment (Bangdkai) [inline build/platform-linux.mk:6]: build/platform-linux.mk

Comment (Bangdkai) [inline build/platform-linux.mk:6]: build/platform-linux.mk

File: build/platform-linux.mk (hunk 11 above)
Comment (Bangdkai) [inline build/platform-linux.mk:6]: build/platform-linux.mk

**Pull request merged**